docs/iot/guides/zigbee.md

```diff
@@ -0,0 +1,321 @@
+
+       /*
+        * It is expected that user has already registered zb_set_event_cb(handle, _zb_event_cb),
+        * because the actual result proving that zigbee device has been disabled
+        * will come in _zb_event_cb().
+        */
+
+       ret = zb_disable(handle);
+       if (ZIGBEE_ERROR_NONE != ret) {
+           printf("zb_disable() - FAILED!!!");
+           return FALSE;
+       }
+
+       return TRUE;
+   }
+   ```
+
+## Form or Disable Zigbee Network Coordinator
+
+1. To form a zigbee network coordinator, you must call `zb_form_network()` API and wait for result in `zb_form_network_cb` callback.
+
+   ```
+   static void _zb_form_network_done_cb(zb_nwk_addr panid, void *user_data)
+   {
+       printf("form_network_done received PANID = 0x%04X\n", panid);
+   }
+
+   int run_form_network(zb_zigbee_h handle)
+   {
+       int ret = ZIGBEE_ERROR_NONE;
+
+       ret = zb_form_network(handle, _zb_form_network_done_cb, NULL);
+       if (ZIGBEE_ERROR_NONE != ret) {
+               msg("zb_form_network() - FAILED!!!");
+               return FALSE;
+       }
+
+       return TRUE;
+   }
+   ```
+
+2. To disable a zigbee network coordinator, you must call `zb_disable_network()` API and wait for result in `zb_disable_network_cb` callback.
+
+   ```
+   static void _zb_disable_network_done_cb(unsigned char ret, void *user_data)
+   {
+       printf("disable_network result received = 0x%02X\n", ret);
+   }
+
+   int run_disable_network(zb_zigbee_h handle)
+   {
+       int ret = ZIGBEE_ERROR_NONE;
+
+       ret = zb_disable_network(handle, _zb_disable_network_done_cb, NULL);
+       if (ZIGBEE_ERROR_NONE != ret) {
+               msg("zb_disable_network() - FAILED!!!");
+               return FALSE;
+       }
+
+       return TRUE;
+   }
+   ```
+
+## Permit Join
+
+Tizen Zigbee Framework on ARTIK works as a Zigbee coordinator, therefore, permission is required to communicate with a new Zigbee device. Zigbee is enabled when a network is created or when a network is re-created. In both the scenarios, you need to make the coordinator to permit joining state.
+
+   ```
+   int permit_join(zb_zigbee_h handle)
+   {
+       int ret = ZIGBEE_ERROR_NONE;
+       unsigned char timeout = 90;
+
+       ret = zb_permit_join(handle, timeout);
+       if (ZIGBEE_ERROR_NONE != ret) {
+               msg("zb_permit_join() - FAILED!!!");
+               return FALSE;
+       }
+
+       return TRUE;
+   }
+   ```
+
+> **Note**
+>
+> After permit setting, enable Zigbee thing search mode using the Zigbee thing device. For more information, see the device manual.
+>
+> If the device is joined, you can see `ZB_ZDP_JOIN_EVENT` in callback registered using `zb_set_event_cb()` API.
+
+## Select Target Device
+To communicate specific Zigbee thing in network, you need to select the target device. If target device is selected, you can communicate and send data to it.
+
+   ```
+    static int run_choose_end_device_list(MManager *mm, struct menu_data *menu)
```

Comment:
I think MManager is not important. it will even cause confusing for users.
So, I suggest you removing MManager or where codes from explicitly (i.e., /zigbee/test/main.c) .